### ISSUE TITLE
Include babel-plugin-transform-strict-mode

### ISSUE THREAD

**Issue by ngbrown** [gajus/babel-preset-es2015-webpack#3]
[`babel-plugin-transform-es2015-modules-commonjs`](https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-es2015-modules-commonjs) included [`babel-plugin-transform-strict-mode`](https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-strict-mode) ([line 122](https://github.com/babel/babel/blob/v6.4.6/packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js#L122)) but is now missing because of the obvious benefit of letting Webpack deal directly with ES6 modules.

Since code in strict mode [can run faster](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode), I think that the `babel-plugin-transform-strict-mode` plugin should be included with `babel-preset-es2015-webpack`.

**Comment by gajus** [comment]
I am all pro strict mode, but what is the reason for including it with `es2015` preset and not as a separate plugin?

What if I prefer `"use strong"` that is added using [`babel-plugin-transform-strong-mode`](https://github.com/gajus/babel-plugin-transform-strong-mode)? Then `"use strict"` is just a noise.

**Comment by ngbrown** [comment]
@gajus, while it's true that someone might want "use strong", or they could add the plugins manually, the normal behavior for `preset-es2015` was to include `"use strict"`, and since the goal of this plugin is to be a derivative of that, but to enable tree-shaking, I think it should continue to operate as closely as possible.  

Also, `"use strong"` is experimental for Chrome only, not for other browsers, while `"use strict"` is respected and provides optimizations on all browsers. `"use strong"` could then be layered on top of that for added enhancements in Chrome.

**Comment by gajus** [comment]
Sounds reasonable. Do you want me to make the change or will you raise a PR? I am happy with either.

**Comment by gajus** [comment]
Published v6.3.14. Thank you.

**Comment by JoyceBabu** [comment]
The strict mode plugin was removed after commit df131861892f3b800786b5cbca61b03805aa1d44

Was this intentional? Will you be adding it back? If not, how can I add `babel-plugin-transform-strict-mode` when using the preset?

**Comment by gajus** [comment]
@JoyceBabu It is not part of the official es2015 preset either.

https://github.com/babel/babel/blob/cae3f60e767dfc3fdd03cc8a0cebff8dd99ca209/packages/babel-preset-es2015/index.js

As far as I remember, other plugins have it as a dependency and therefore it is included eventually. Just see the output of Babel. It should include `"use strict";` header.

**Comment by JoyceBabu** [comment]
The final output did not have the global `"use strict";` header. When I was using the `es2015` preset, all the `"use strict";` headers in individual functions were replaced with a global declaration. But it is not happening with `es2015-webpack` preset.

I tried adding `transform-strict-mode` plugin in addition to the `es2015-webpack` preset and then the final output had only a single global strict declaration.
